src/main/java/org/jabref/gui/mergeentries/newmergedialog/FieldRowViewModel.java

```diff
@@ -282,4 +288,49 @@ public void redo() throws CannotRedoException {
             setRightFieldValue(mergedFields);
         }
     }
+
+    /**
+     * Method for selecting the 'Better' year value.
+     * If the local year is out of a reasonable range (e.g., before 1800 or 100 years after current year as determined by the System Clock) or differs from the DOI year by more than 10 years, it will choose the more recent year out of the two.
+     */
+    public void autoSelectBetterValue() {
+        String field_1 = getField().getDisplayName();
+        if (field_1 == null) {
+            return;
+        }
+        field_1 = field_1.trim().toLowerCase();
```

Comment:
What is this? When can dispalyname be null? Why `toLowerCase`? - we also have `.getName` - not sure what this code aims vor.